lib/ConstExtract/ConstExtract.cpp

```diff
@@ -239,3 +242,22 @@ bool writeAsJSONToFile(const std::vector<ConstValueTypeInfo> &ConstValueInfos,
 }
 
 } // namespace swift
+
+#define SWIFT_TYPEID_ZONE ClangImporter
```

Comment:
Potential copy-pasta?
```suggestion
#define SWIFT_TYPEID_ZONE ConstExtract
```

Reply:
Oof. Thanks for catching.